.github/workflows/e2e-main.yaml

```diff
@@ -97,33 +104,20 @@ jobs:
       - name: Build Podman Desktop for E2E tests
         working-directory: ./podman-desktop
         run: |
-          yarn --frozen-lockfile
-          yarn test:e2e:build
-
-      - name: Get yarn cache directory path
-        working-directory: ${{ env.REPOSITORY }}
-        id: yarn-cache-dir-path
-        run: echo "dir=$(yarn cache dir)" >> ${GITHUB_OUTPUT}
-
-      - uses: actions/cache@v4
-        id: yarn-cache
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
+          pnpm install
+          pnpm test:e2e:build
 
-      - name: Execute yarn in Red Hat Account Extension
+      - name: Execute pnpm in Red Hat Account Extension
         working-directory: ${{ env.REPOSITORY }}
         run: |
-          yarn add -D @podman-desktop/tests-playwright@next
-          yarn --frozen-lockfile
+          pnpm install -D @podman-desktop/tests-playwright@1.13.0-202408291153-ccbdd9b
```

Comment:
I do no think we can have specific version checksum here. we need `next` as it automatically adds latest version (nightly) in the package.json and installs it afterwards.